tree/dataframe/test/dataframe_snapshot.cxx

```diff
@@ -897,7 +897,7 @@ TEST(RDFSnapshotMore, MissingSizeBranch)
    ROOT::RDataFrame df("t", inFile);
 
    // fully typed Snapshot call throws
-   EXPECT_THROW(df.Snapshot<ROOT::RVecF>("t", "NeverWrittenOut.root", {"vec"}), std::runtime_error);
+   EXPECT_THROW(df.Snapshot<ROOT::RVecF>("t", outFile, {"vec"}), std::runtime_error);
```

Comment:
I have the very stupid question how to try it locally - `ctest -R "(dataframe_snapshot)"  --verbose -j` did not find any tests.

Isn't the/a problem here that we never unlink the outfile. E.g. missing: `gSystem->Unlink(outFile);`. How/Why would a variable guarantee a deletion?

Reply:
(I can do `ctest -R "(snapshot)" --verbose -j` but then there are only 3 tests, which do not include this one.)

Reply:
> Isn't the/a problem here that we never unlink the outfile

absolutely, thanks! fixed.

> I can do ctest -R "(snapshot)" --verbose -j but then there are only 3 tests, which do not include this one

that must be a problem with your build

Reply:
I agree with both statements. I fixed my ROOT. A just realized however that there are other places where files are not removed. Before executing the snapshot tests:
```
ls tree/dataframe/test/*.root --> is empty
```
After running the tests:
```
 ls tree/dataframe/test/*.root                                                                                                                                                                                                                    
tree/dataframe/test/Snapshot_aliases.root*             tree/dataframe/test/snap_update_sametreenames.root*     tree/dataframe/test/test_snapshotRVecoutMT.root*        tree/dataframe/test/test_snapshotRVecoutST.root*
tree/dataframe/test/snapshot_reshuffled_friends.root*  tree/dataframe/test/test_snapshotRVecoutMTJitted.root*  tree/dataframe/test/test_snapshotRVecoutSTJitted.root*
```
I can take care of that.

Reply:
yes, but unrelated to this PR, and absolutely feel free to jump on this :D

Reply:
(should be quick xD)